projects/storefrontlib/src/layout/layout.module.ts

```diff
@@ -1,13 +1,12 @@
 import { NgModule } from '@angular/core';
 import { Config } from '@spartacus/core';
 import { OutletRefModule } from '../cms-structure/outlet/outlet-ref/outlet-ref.module';
+import { SkipLinkModule } from './a11y';
```

Comment:
you should explicitly import from the barrel file; we often get into trouble during build